bullet-featherstone/src/WorldFeatures.cc

```diff
@@ -52,6 +52,51 @@ WorldFeatures::LinearVectorType WorldFeatures::GetWorldGravity(
     return WorldFeatures::LinearVectorType(0, 0, 0);
   }
 }
+
+/////////////////////////////////////////////////
+void WorldFeatures::SetWorldSolver(const Identity &,
+    const std::string &_solver)
+{
+  // bullet-featherstone implementation uses btMultiBodyDynamicsWorld
+  // so currently only supports BT_MULTIBODY_SOLVER
+  if (_solver != "MultiBodySolver")
+  {
+    gzwarn << "bullet-featherstone implementation currently only supports "
+           << "MultiBodySolver (BT_MULTIBODY_SOLVER)" << std::endl;
+  }
+}
+
+/////////////////////////////////////////////////
+const std::string &WorldFeatures::GetWorldSolver(const Identity &) const
+{
+  // bullet-featherstone implementation uses btMultiBodyDynamicsWorld
+  // so currently only supports BT_MULTIBODY_SOLVER
+  static std::string solverType = "MultiBodySolver";
+  return solverType;
+}
+
+/////////////////////////////////////////////////
+void WorldFeatures::SetWorldSolverIterations(const Identity &_id,
+    std::size_t _iterations)
```

Comment:
include `<cstddef>`

Reply:
added in header file. [adc1b3d](https://github.com/gazebosim/gz-physics/pull/609/commits/adc1b3dac83b67461cf0a67c8566540956782307)